cloudfoundry/init/env.properties

```diff
@@ -21,3 +21,7 @@ LOG_SERVICE_NAME=
 CONFIG_SERVER_SERVICE_NAME=p-config-server
 CONFIG_SERVER_PLAN_NAME=trial
 TRUST_CERTS=api.run.pivotal.io
+SCHEDULES_SERVICE_NAME=scheduler-for-pcf
+SCHEDULES_PLAN_NAME=standard
+SCHEDULES_INSTANCE_NAME=atscheduler
+SCHEDULES_URL=https://scheduler.sys.needles.cf-app.com
```

Comment:
Please remove the explicit names here. We can have them configured in the CI plan instead.